apps/vr-tests-react-components/src/stories/Toast/Toast.stories.tsx

```diff
@@ -136,3 +136,86 @@ export const TitleOnly = () => {
     </>
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 6 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=28354&commitId=36148132dd8ce077ce9580fe546a0450c2047d1d&env=prod)

✅ There was 6 screenshots added, 0 screenshots removed, 1954 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 6 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Toast.Full Toast Inverted.Toast visible.chromium.png|0| Added|
|Toast.Full Toast Inverted.chromium.png|0| Added|
|Toast.Title Only Inverted.Toast visible.chromium.png|0| Added|
|Toast.Title Only Inverted.chromium.png|0| Added|
|Toast.Without Subtitle Inverted.Toast visible.chromium.png|0| Added|
|Toast.Without Subtitle Inverted.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>